tools/gorm/compare_migrations.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -e
+
+# This script can be used to determine whether there will be any gorm db schema changes from a git branch
+#
+# It assumes that the input data source has the db schema from the master branch
+# It generates the gorm auto-migration SQL statements on the master branch, then compares these statements to the
+# SQL statements on the input branch and prints any diff
+# It does not execute any migrations on the db
+
+git_branch=$1
+data_source=$2
+
+git checkout master
```

Comment:
will this be configurable? (the master branch is unstable so I guess this will probably not reflect the actual migration that'll happen)